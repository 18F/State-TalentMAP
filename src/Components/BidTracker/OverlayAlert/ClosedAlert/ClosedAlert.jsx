import React from 'react';

const ClosedAlert = () => (
  <div className="bid-tracker-alert-container bid-tracker-alert-container--closed">
    <div className="top-text">
      This bid has been closed
    </div>
  </div>
);

export default ClosedAlert;
