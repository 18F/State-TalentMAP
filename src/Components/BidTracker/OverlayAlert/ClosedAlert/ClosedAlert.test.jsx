import { shallow } from 'enzyme';
import React from 'react';
import toJSON from 'enzyme-to-json';
import ClosedAlert from './ClosedAlert';

describe('ClosedAlertComponent', () => {
  it('is defined', () => {
    const wrapper = shallow(
      <ClosedAlert />,
    );
    expect(wrapper).toBeDefined();
  });

  it('matches snapshot', () => {
    const wrapper = shallow(
      <ClosedAlert />,
    );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
