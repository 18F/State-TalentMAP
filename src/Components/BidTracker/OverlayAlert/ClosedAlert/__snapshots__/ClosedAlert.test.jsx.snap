// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClosedAlertComponent matches snapshot 1`] = `
<div
  className="bid-tracker-alert-container bid-tracker-alert-container--closed"
>
  <div
    className="top-text"
  >
    This bid has been closed
  </div>
</div>
`;
